buildkite/resource_pipeline.go

```diff
@@ -15,46 +15,43 @@ const defaultSteps = `steps:
   command: buildkite-agent pipeline upload`
 
 // PipelineNode represents a pipeline as returned from the GraphQL API
+type Cluster struct {
+	ID graphql.String
+}
+type Repository struct {
+	URL graphql.String
+}
+type Steps struct {
+	YAML graphql.String
+}
```

Comment:
Is there a reason to _not_ use the `bindings` option in Genqlient for these?

Reply:
Oh yeah these are for the shurcool client. So I wasn't too concerned with doing it a nice way since we'll get rid of it shortly. I just couldn't get it working unless I had a dedicated struct to also use elsewhere to map between the shurcool and genqlient types

Reply:
Yeah, OK, my understanding of `bindings` is that it'll create the type for you when you run the generation, but it's a non-blocker.